ui/src/Components/FeedbackType.js

```diff
@@ -0,0 +1,4 @@
+export const FeedbackType = {
+  My: 1,
+  Employee: 2,
```

Comment:
what is 'MY'? Change the names to something meaningful

Reply:
done